src/NuGet.Core/NuGet.Packaging/NuGet.Packaging.csproj

```diff
@@ -49,6 +49,8 @@
 
   <ItemGroup Condition=" '$(IsCore)' == 'true' ">
     <PackageReference Include="System.Dynamic.Runtime" Version="$(SystemPackagesVersion)" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="5.0.0-alpha1.19473.1" />
```

Comment:
This PR change is fragile.  You have a package version `5.0.0-alpha1.19473.1` hard-coded in 3 places and more versions hard-coded in `TempPatchingDepsJsonForNewlyAddedDlls()`.

Please create an issue for replacing the temporary workaround with a permanent, more robust fix.  Then link that issue with this PR to create some continuity for whoever (even you) who may have to implement a permanent fix.